packages/ethereum/ethereum-types/src/consensus/bootstrap.rs

```diff
@@ -19,5 +16,5 @@ pub struct LightClientBootstrap {
     /// The current sync committee
     pub current_sync_committee: SyncCommittee,
     /// The branch of the current sync committee
-    pub current_sync_committee_branch: [B256; CURRENT_SYNC_COMMITTEE_BRANCH_DEPTH],
+    pub current_sync_committee_branch: Vec<B256>,
```

Comment:
Add the todo item here as well. Also, maybe we can add a validate basic here

Reply:
It's not really used in the light client directly. We use it to construct the initial consensus state only. We could potentially add some more verification logic in the consensus state that we can also check in the update and init to be fully sure?